organizations/v0/tests/test_views.py

```diff
@@ -0,0 +1,67 @@
+"""
+Organizations Views Test Cases.
+"""
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+
+from organizations.tests.factories import UserFactory, OrganizationFactory
+
+
+DRF_DATE_FORMAT = u'%Y-%m-%dT%H:%M:%S.%f'
```

Comment:
As was done for credentials, set [`DATETIME_FORMAT`](http://www.django-rest-framework.org/api-guide/settings/#datetime_format) and reference it.